src/test/java/uk/ac/ebi/atlas/experimentpage/JsonAnatomogramCellTypeHeatmapControllerWIT.java

```diff
@@ -0,0 +1,13 @@
+package uk.ac.ebi.atlas.experimentpage;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JsonAnatomogramCellTypeHeatmapControllerWIT {
+
+    @Test
+    void foo() {
+        assertThat(true).isFalse();
```

Comment:
I was looking for a test case to test that relevant endpoint. I am confused here, what we are testing here @alfonsomunozpomer.

Reply:
Oh, I configured IDEA with that template, so that whenever I create new classes with their corresponding tests, by default it will always fail. In that way, if I forget to add a test it will always fail. In this case this was from a class `JsonAnatomogramCellTypeHeatmapController` that ended being not relevant and I removed the source class but the test remained.